paper/paper.md

```diff
@@ -68,16 +68,25 @@ spent on data management should be as small as possible. The use of `secuTrialR`
 necessary for data management, enables swift quantitative analyses through preimplemented functionalities and, most importantly,
 standardizes the interaction with data exports from secuTrial, thus allowing robust and reproducible science.
 
+While some CDMSs provide APIs (e.g. REDCap [@Harris2009; @Harris2019]) or ODBC connections (e.g. 
```

Comment:
Rather just CDMS?

What is an ODBC? Maybe write the abbreviation out.

Reply:
done